src/LSST/control/Model.h

```diff
@@ -62,6 +66,11 @@ class Model {
     /// VI-PH retrieveStateVI  // calls breaks out State and currentStateEnum and returns
     std::shared_ptr<State> getCurrentState();
 
+    /// Go to safe mode.
+    /// @param note - Description of reason for safe mode.
+    /// @return true if the system was not already trying to reach.
+    bool goToSafeMode(std::string const& note);

```

Comment:
In the current control system, the safe mode is defined to be cut off the powers of motor and communication. You could explain this here to clarify what is the "safe mode".